LondonBritishLibrary/orient/BLorient12114A.xml

```diff
@@ -0,0 +1,352 @@
+                                <locus from="150r" to="159v"/>
+                                <title ref="LIT2509Weddas" type="incomplete"/>
+                                <note>Arranged by the days of the week, with Monday's lesson first. The end of Monday's lesson is missing together with
+                                    the beginning of the text for Tuesday, because one leaf is missing after <ref target="#150"/>. The text ends ex abrupto, 
+                                    because probably two leaves are lacking missing.</note>
+                                <explicit xml:lang="gez">አንቲ፡ ውእቱ፡ መሶበ፡ ወ<supplied reason="lost" resp="PRS8999Strelcyn">ርቅ፡</supplied></explicit>
+                            </msItem>
+                            <msItem xml:id="p1_i1.4">
+                                <locus from="160ra" to="163vb"/>
+                                <title ref="LIT1113Anqasa" type="incomplete"/>
+                                <note>The beginning and the end are missing.</note>
+                                <incipit xml:lang="gez"><supplied reason="lost" resp="PRS8999Strelcyn">ወ</supplied>ልድ፡ ወምጽላል፡ ዘቅዱስ፡ መንፈስ።
+                                    ኦቡርክት፡ እምኵሉ፡ ፍጥረት፡ አንቲ፡ ውእቱ፡ ህየንተ፡ አርያም፡ ዘበሰማያት፡ </incipit>
+                                <explicit xml:lang="gez">ወታስተስርዪ፡ ኃጢአተ፡ ሕዝብኪ፡ ተበውሃ፡ ለኪ፡ እም፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ከመ፡ ትኩኒ፡ ተንከተመ፡ 
+                                    ለው<supplied reason="lost" resp="PRS8999Strelcyn">ሉደ፡ ሰብእ፡</supplied></explicit>
+                            </msItem>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">163</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>142</height>
+                                        <width>138</width>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="18"><locus from="1r" to="149v"/></layout>
+                                <layout columns="2" writtenLines="18"><locus from="150r" to="163v"/></layout>
+                            </layoutDesc>    
+                        </objectDesc> 
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Rather careful handwriting.</desc>
+                                <date notBefore="1600" notAfter="1800" resp="PRS8999Strelcyn"/>
```

Comment:
Do we have any guidelines as to how we write the dating? Strelcyn might probably mean "17th or 18th century" (what I  personally indicate 17th/18th century). The dating embracing the span of 200 years is a little odd (in his catalogue mostly without arguments anyway).